XamarinCommunityToolkit/Behaviors/ConfirmPasswordBehavior.cs

```diff
@@ -0,0 +1,22 @@
+using Xamarin.Forms;
+
+namespace Microsoft.Toolkit.Xamarin.Forms.Behaviors
+{
+    public class ConfirmPasswordBehavior : ValidationBehavior
+    {
+        public static readonly BindableProperty ComparedTextProperty =
+            BindableProperty.Create(nameof(ComparedText), typeof(string),
+                typeof(ConfirmPasswordBehavior), string.Empty);
```

Comment:
the default value should be "null" I guess
```suggestion
                typeof(ConfirmPasswordBehavior), string.Empty);
```

Reply:
sorry you mean here 
        public static readonly BindableProperty ComparedTextProperty =
            BindableProperty.Create(nameof(ComparedText), typeof(string),
                typeof(ConfirmPasswordBehavior));
Instaed of:
        public static readonly BindableProperty ComparedTextProperty =
            BindableProperty.Create(nameof(ComparedText), typeof(string),
                typeof(ConfirmPasswordBehavior), string.Empty);

Reply:
Yes, the default value is null *by default* 👍